### ISSUE TITLE
Audio decoding issue breaks waveform

### ISSUE THREAD

**Issue by lkiesow** [elan-ev/opencast-editor#254]
On [Olaf's test event on pyca.opencast.org](https://editor.opencast.org/?mediaPackageId=601ade55-625a-4f12-bc06-71d0c57d841d) I get the error:

```
DOMException: The buffer passed to decodeAudioData contains an unknown content type.
```

…and the user interface keeps saying „Generating Waveform“:

![Screenshot from 2021-04-08 18-57-10](https://user-images.githubusercontent.com/1008395/114067103-bab9f000-989c-11eb-971a-c10bc0f516a4.png)

This may be caused by the event being processed with the “Fast Testing Workflow”, but if there is an error there should just be no waveform as a fallback and users should still be able to cut the event.

**Comment by Arnei** [comment]
The PR #236 should catch that exception and then display a message like "Could not generate waveform". That exception can for example occur when the video has no audio.

Being able to cut the event should already be independent of whether a waveform is generated or not. That cause of that problem probably lies in the issue described in #253

**Comment by oas777** [comment]
As advised by Lars, I repeated testing this with an MP4 (the other video file uploaded was a WEBM):
- waveform still not showing (this is Firefox on Windwos)
- time is displayed correctly
https://editor.opencast.org/?mediaPackageId=1bfc63f8-eb0d-4da7-b00b-02cb2867611d

**Comment by oas777** [comment]
![waveform_firefox](https://user-images.githubusercontent.com/1752342/114156985-6f96f000-9923-11eb-9d15-0e666f2223ba.PNG)

**Comment by lkiesow** [comment]
Likely already resolved. Needs another test.

**Comment by Arnei** [comment]
Tested it again. The problem here was the video having no audio, and the resulting exception not being handled, which was solved by #236. 
Furthermore, waveform generation did not work in Safari at all, which was fixed by #256. Therefore I'll consider this issue closed.
